src/core_modules/capture-core/components/DataEntries/SingleEventRegistrationEntry/SingleEventRegistrationEntry.types.js

```diff
@@ -1,24 +1,26 @@
 // @flow
-export type Props = {|
+export type StateProps = {|
+    ready: boolean,
     showAddRelationship: boolean,
     eventAccess: {|
         read: boolean,
         write: boolean,
     |},
-|};
-
-export type StateProps = {|
-    ready: boolean,
-    ...Props,
 |}
 
 export type ContainerProps = {|
     id: string,
     selectedScopeId: string,
 |};
 
-export type DispatchProps = {|
-    onCancel: () => void,
+export type Props = {|
+    ...ContainerProps,
+    ...StateProps,
 |};
 
+export type DispatchProps = () => {||};
```

Comment:
The `DispatchProps` type is not used anywhere. Is it possible to import it and use it instead of `{||}` in [mergeProps](https://github.com/dhis2/capture-app/blob/DHIS2-17519-app-crashing-when-opening-new-event-from-view-event/src/core_modules/capture-core/components/DataEntries/SingleEventRegistrationEntry/SingleEventRegistrationEntry.container.js#L30) and [compose](https://github.com/dhis2/capture-app/blob/DHIS2-17519-app-crashing-when-opening-new-event-from-view-event/src/core_modules/capture-core/components/DataEntries/SingleEventRegistrationEntry/SingleEventRegistrationEntry.container.js#L68)? Thanks!